### DISCUSSION TITLE
When will version 6.3.1 be released?

### DISCUSSION THREAD

**Discussion by TheNerka** [symfony/symfony#50510] [Q&A]
Hello, when will version 6.3.1 be released? needs this fix https://github.com/symfony/symfony/pull/50498 

neither command is working now due to this error :(

or how to pull this commit?

**Comment by WebMamba** [comment]
Here is the release process doc: https://symfony.com/doc/current/contributing/community/releases.html.
So the 6.3.1 should be released in one month. 😁
But I think removing the cache for your issue should do the trick.
